cdc/log.cc

```diff
@@ -106,8 +106,13 @@ class cdc::cdc_service::impl : service::migration_listener::empty_listener {
         // or if cdc is on now unconditionally, since then any actual base schema changes will affect the column 
         // etc.
         if (was_cdc || is_cdc) {
-            auto logname = log_name(old_schema.cf_name());
             auto& db = _ctxt._proxy.get_db().local();
+
+            if (!was_cdc) {
+                check_that_cdc_log_table_does_not_exist(db, new_schema, log_name(new_schema.cf_name()));
+            }
+
+            auto logname = log_name(old_schema.cf_name());
```

Comment:
supper nitpick: move this above check_that_cdc... and reuse logname in the call of check_that_cdc...